worknenjoy/gitpay
# Welcome user component

## Created a welcome user component 

> Added a new component to session folder called welcome-user.js

## Added a call to welcomeUser in login-page.js and a new component named WelcomeUser

- login-page.js, welcome-user.js, welcome1.png, welcome2.png, welcome3.png

## 534

> Closes #534 

## Impacted Area

> Welcome user popup

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create user
- Once user is created successfully, a welcome dialogue page appears

## Before
> <img width="1110" alt="Screenshot 2020-08-30 at 16 20 39" src="https://user-images.githubusercontent.com/41532077/91657184-c52d3380-eadc-11ea-99fc-ce796d9695c2.png">

## After
## First step doesn't have previous
<img width="584" alt="Screenshot 2020-09-05 at 12 43 14" src="https://user-images.githubusercontent.com/41532077/92300300-50904400-ef77-11ea-92f3-7ebbc5101d30.png">
## Welcome user appears only first time on welcome page
<img width="1630" alt="Screenshot 2020-09-05 at 12 59 55" src="https://user-images.githubusercontent.com/41532077/92300332-b7156200-ef77-11ea-97ad-d112d6dcd158.png">


## Last step (added finish button that closes the dialogue box)
<img width="726" alt="Screenshot 2020-09-05 at 12 55 43" src="https://user-images.githubusercontent.com/41532077/92300280-2fc7ee80-ef77-11ea-8649-c156e37175a4.png">
